### PR TITLE
Use status-im/whisper module with all changes that we have

### PR DESCRIPTION
**Pull request by dshulyak**
Using forked whisper version from https://github.com/status-im/whisper

### CODE REVIEW

**Comment by adambabik** [comment]
https://github.com/status-im/status-go/pull/1184 is merged.

**Comment by dshulyak** [comment]
@adambabik thanks, updated

File: node/node.go [hunk 1]
```diff
@@ -271,17 +271,18 @@ func activateShhService(stack *node.Node, config *params.NodeConfig, db *leveldb
 		whisperServiceConfig := &whisper.Config{
 			MaxMessageSize:     whisper.DefaultMaxMessageSize,
 			MinimumAcceptedPOW: 0.001,
-			TimeSource:         time.Now,
```
Comment (adambabik) [inline node/node.go:274]: Does it mean that `TimeSource` in config is unused and can be removed?

**Review by adambabik — APPROVED** [review]

File: node/node.go (hunk 1 above)
Comment (dshulyak) [reply node/node.go:274]: in what config? i removed TimeSource from whisper.Config and now it is set with separate method. it was simpler to integrate with tests this way. as for NTPSyncEnabled parameter from status config - it is still used and doesn't need to be removed

Comment (adambabik) [reply node/node.go:274]: > i removed TimeSource from whisper.Config

That's what I meant :)

**Comment by divan** [comment]
I still see some references to the previous import:
```
$ rg -c "go-ethereum/whisper"
Gopkg.lock:1
peers/peerpool_test.go:1
vendor/github.com/ethereum/go-ethereum/mobile/types.go:1
vendor/github.com/ethereum/go-ethereum/mobile/geth.go:1
vendor/github.com/ethereum/go-ethereum/mobile/shhclient.go:2
vendor/github.com/ethereum/go-ethereum/cmd/geth/config.go:1
vendor/github.com/ethereum/go-ethereum/cmd/wnode/main.go:2
vendor/github.com/ethereum/go-ethereum/cmd/utils/flags.go:1
vendor/github.com/ethereum/go-ethereum/swarm/pss/types.go:1
vendor/github.com/ethereum/go-ethereum/swarm/pss/pss.go:1
vendor/github.com/ethereum/go-ethereum/whisper/shhclient/client.go:1
vendor/github.com/ethereum/go-ethereum/whisper/mailserver/mailserver.go:1
```

`peers/peerpool_test.go`, probably, the only which should be fixed, but those imports in `vendor/` also theoretically can have impact. I think we need to finally solve pruning issue for geth, as those unused packages bring confusion and uncertainity sometimes.

**Comment by dshulyak** [comment]
thanks. peers_test was using whisper just for the sake of establishing connection using known protocol. removed that import

**Review by divan — APPROVED** [review]

**Pull request merged**